extensions/wikia/ARecoveryEngine/js/PageFair/pageFair.js

```diff
@@ -33,10 +33,10 @@ define('ext.wikia.aRecoveryEngine.recovery.pageFair', [
 		return isBlocking;
 	}
 
-	function addMarker(slotId) {
-		var slot = document.getElementById(slotId);
-
-		slot.setAttribute('adonis-marker', '');
+	function addMarker(slotElement) {
+		if (slotElement) {
```

Comment:
Is the `if` really needed? We always [create an element](https://github.com/Wikia/app/pull/12808/files#diff-d0456a56fa8978685b777a3c10aaf5abR88) before we call `addMarker()` with it, don't we?